app/routes/photos/components/GalleryPictureModal.js

```diff
@@ -194,7 +193,7 @@ export default class GalleryPictureModal extends Component<Props, State> {
     }
   };
 
-  handleSwipe = ({ dir }: { dir: RIGHT | LEFT }) => {
+  handleSwipe = ({ dir }: { dir: typeof RIGHT | typeof LEFT }) => {
```

Comment:
Hmm, this seems a bit weird 🤔 . Can you not treat this as an enum anymore? I'm guessing this now is identical to `string | string`.

Reply:
Yeah, types can be a bit strange, but I think this is the best way to do it (and it works).

![Screenshot from 2021-01-06 10-41-41](https://user-images.githubusercontent.com/1467188/103753811-cb439b80-500b-11eb-98c5-02b85e114d5a.png)

(197 works, 198 breaks)

Reply:
Ah. I guess I am mistaken then. Awesome

Reply:
lol. You were right. This is too early I guess.... Changed to string now :sweat_smile: